docs/publish/host-an-office-add-in-on-microsoft-azure.md

```diff
@@ -1,225 +1,180 @@
 
+## Step 1: Create a shared folder to host your add-in XML manifest file
 
-1. Open File Explorer (or Windows Explorer if you're using Windows 7 or an earlier version of Windows) on your development computer.
+1. Open File Explorer on your development computer.
     
-2. Right-click the C:\ drive, and then choose  **New** > **Folder**.
+2. Right-click the C:\ drive and then choose **New** > **Folder**.
     
 3. Name the new folder AddinManifests.
     
-4. Right-click the AddinManifests folder, and then choose  **Share with** > **Specific people**.
+4. Right-click the AddinManifests folder and then choose **Share with** > **Specific people**.
     
-5. In  **File Sharing**, choose the drop-down arrow and then choose  **Everyone** > **Add** > **Share**.
+5. In **File Sharing**, choose the drop-down arrow and then choose **Everyone** > **Add** > **Share**.
     
+> **Note:** In this walkthrough, you're using a local file share as a trusted catalog where you'll store the add-in XML manifest file. In a real-world scenario, you might instead choose to [deploy the XML manifest file to a SharePoint catalog](../publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md) or [publish the add-in to the Office Store](https://dev.office.com/officestore/docs/submit-to-the-office-store).
 
-## Step 2: Add the file share to the Trusted Add-ins catalog so that Office client applications will trust the location where you install Office Add-ins
+## Step 2: Add the file share to the Trusted Add-ins catalog
 
+1.  Start Word 2016 and create a document.
 
-
-1.  Start Word 2013 and create a document. (Although we're using Word 2013 in this example, you could use any Office application that supports Office Add-ins like Excel, Outlook, PowerPoint, or Project 2013.)
+    >**Note:** Although this example uses Word 2016, you can use any Office application that supports Office Add-ins such as Excel, Outlook, PowerPoint, or Project 2016.
     
 2.  Choose **File** > **Options**.
     
-3.  In **Word Options**, choose  **Trust Center**, and then choose  **Trust Center Setting**. 
-    
-4.  In the **Trust Center**, click  **Trusted Add-in Catalogs**. Enter the universal naming convention (UNC) path for the file share you created earlier as the  **Catalog URL**. For example, \\YourMachineName\AddinManifests. Then choose  **Add catalog**. 
+3.  In the **Word Options** dialog, choose **Trust Center** and then choose **Trust Center Settings**. 
     
-5. Select the check box for  **Show in Menu**. When you store an add-in XML manifest file on a share that is a trusted add-in catalog, the add-in appears under  **Shared Folder** in the **Office Add-ins** dialog box.
+4.  In the **Trust Center** dialog, click **Trusted Add-in Catalogs**. Enter the universal naming convention (UNC) path for the file share you created earlier as the **Catalog URL** (for example, \\\YourMachineName\AddinManifests), and then choose **Add catalog**. 
```

Comment:
Minor point - we use "dialog box" instead of "dialog". And the "click" should be "choose".